tmp/pids/server.pid

```diff
@@ -0,0 +1 @@
+1510
```

Comment:
@Mywe4ka Please add all /tmp files to .gitignore.
And for the future go to /files submenu on your PR and check all items you've added.